R/create_netcdf_output.R

```diff
@@ -86,7 +85,7 @@ create_netcdf_output <- function(output_lists, folder = ".", model, out_time,
       
       lengths <- lapply(output_lists[[i]], ncol) # Extract ncols in each output
       lon_list <- which.max(lengths) # Select largest depths
-      deps <- rLakeAnalyzer::get.offsets(output_lists[[i]][[lon_list]]) # Extract depths
```

Comment:
just curious why this change was made? It doesn't hurt to use the `::` and makes it cleared to future devs (or yourself) where this function comes from.

Reply:
We copied the get_offsets function to LER (helpers.R script). This was done to reduce the amount of dependencies. get_offsets was the only function from rLakeAnalyzer that was used. 

In reality, rLakeAnalyzer still needs to be installed, as gotmtools relies on it, but we figured having less dependencies in LER itself was better, especially if a rather simple function such as get.offsets was the cause.

Reply:
oh, ok. Sounds good. I didn't see that change. Makes sense!